recipes/rospkg/recipe/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright <YEAR> <COPYRIGHT HOLDER>
```

Comment:
Is this the acutal LICENSE file?

Reply:
The problem is that they don't have any license file. The license is only indicated in the setup.py. should I select that as license file?

Reply:
Ok, I copied the header of one of the files of rospkg now. Seems more appropriate.